compiler/src/dotty/tools/dotc/core/tasty/TastyFormat.scala

```diff
@@ -299,6 +300,7 @@ object TastyFormat {
   final val DEFAULTparameterized = 30
   final val STABLE = 31
   final val MACRO = 32
+  final val OPAQUE = 33
```

Comment:
Does this deserve a version bump in the tasty format?

Reply:
You are right, it should be a minor version bump.